server/i18n/en.json

```diff
@@ -2448,6 +2448,14 @@
     "id": "api.post.check_for_out_of_channel_mentions.message.one",
     "translation": "@{{.Username}} did not get notified by this mention because they are not in the channel."
   },
+  {
+    "id": "api.post.check_for_out_of_team_mentions.message.multiple",
+    "translation": "@{{.Usernames}} and @{{.LastUsername}} did not get notified by this mention because they are not a member of this team."
```

Comment:
cc @cwarnermm

Reply:
I believe this would read as: "John and Smith did not get notified by this mention because they are not members of this team."

What's intended here? A single user's first/last name, or potentially multiple users? If multiple is both first and last name expected?

Reply:
> @alice.matthews, @amanda.little and @calebmatterbridge did not get notified by this mention because they are not a member of this team.

Multiple users, not first-name last-name

Reply:
Thanks, @calebroseland! I agree with your recommended updates.